source/ui/BioXAS/BioXASSidePersistentView.cpp

```diff
@@ -61,7 +72,7 @@ BioXASSidePersistentView::~BioXASSidePersistentView()
 void BioXASSidePersistentView::onCalibrateEnergyButtonClicked()
 {
 	bool inputOK = false;
-	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), 0, 10000000, 1, &inputOK, Qt::Sheet);
+	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), -10000000, 10000000, 1, &inputOK, Qt::Sheet);
```

Comment:
I knew that BioXAS was trying to push the limits, but negative energy is pretty bold ;).  Out of curiosity, how is it possible to have a negative energy?  Otherwise, the change itself is okay.